open-metadata-implementation/access-services/data-platform/data-platform-server/src/main/java/org/odpi/openmetadata/accessservices/dataplatform/admin/DataPlatformAdmin.java

```diff
@@ -88,38 +85,16 @@ public void initialize(AccessServiceConfig accessServiceConfig,
             this.serverName=instance.getServerName();
 
             String inTopicName = getTopicName(accessServiceConfig.getAccessServiceInTopic());
-            String outTopicName = getTopicName(accessServiceConfig.getAccessServiceOutTopic());
             dataPlatformInTopicConnector = initializeDataPlatformTopicConnector(accessServiceConfig.getAccessServiceInTopic());
-            dataPlatformOutTopicConnector = initializeDataPlatformTopicConnector(accessServiceConfig.getAccessServiceOutTopic());
 
             OMEntityDao omEntityDao = new OMEntityDao(enterpriseConnector, supportedZones, auditLog);
-            EventPublisher eventPublisher = null;
-
-            if (enterpriseOMRSTopicConnector != null) {
-                auditCode = DataPlatformAuditCode.SERVICE_REGISTERED_WITH_ENTERPRISE_TOPIC;
-                auditLog.logRecord(actionDescription,
-                        auditCode.getLogMessageId(),
-                        auditCode.getSeverity(),
-                        auditCode.getFormattedLogMessage(serverName),
-                        null,
-                        auditCode.getSystemAction(),
-                        auditCode.getUserAction());
-
-                eventPublisher = new EventPublisher(dataPlatformOutTopicConnector, auditLog);
-                DataPlatformEnterpriseOmrsEventListener dataPlatformEnterpriseOmrsEventListener = new DataPlatformEnterpriseOmrsEventListener(eventPublisher, auditLog);
-                enterpriseOMRSTopicConnector.registerListener(dataPlatformEnterpriseOmrsEventListener);
```

Comment:
Is DataPlatform not listening on cohort topic anymore?

Reply:
For now I don't see any use case that DP is getting any events from cohort. Maybe I will add it when the functionality is needed.